__tests__/product/product.mutation.test.js

```diff
@@ -238,61 +233,51 @@ describe('Product mutations', () => {
     const receivedProduct = getProduct.data.getProductById;
     expect(receivedProduct).toBeDefined();
     expect(receivedProduct).toHaveProperty('name', [
-      { __typename: 'Language', value: 'Very Coool Baggy' },
-      { __typename: 'Language', value: 'ДУЖЕ СУПЕРСЬКИЙ Рюкзачечок' },
+      { value: 'Very Coool Baggy' },
+      { value: 'ДУЖЕ СУПЕРСЬКИЙ Рюкзачечок' },
     ]);
     expect(receivedProduct).toHaveProperty('description', [
-      { __typename: 'Language', value: 'Baggy is so cool' },
-      { __typename: 'Language', value: 'Рюкзачечок - супер кльовий))' },
+      { value: 'Baggy is so cool' },
```

Comment:
Please, don't hardcode values which you assert, instead use properties of the object that you have used for the creation of the product. It will reduce the required changes if some data of the product is going to be changed

Reply:
solved